adam/experiment/log_experiment.py

```diff
@@ -28,7 +28,7 @@ def main(params: Parameters) -> None:
     logger = LearningProgressHtmlLogger.create_logger(
         output_dir=output_dir,
         experiment_name=experiment_name,
-        curriculum_name="m6_curriculum",
+        curriculum_name="index",
```

Comment:
This seems somewhat of an abuse of the parameter - would it be better to folder the curriculum name into the directory name (or just not take the curriculum name at all, and let the user put it in the experiment name?), and then have the code write to an `index.html`?